src/lib.rs

```diff
@@ -112,6 +117,12 @@ fn get_values(
                     }),
                 },
                 DataType::Empty => result_row.push(CellValue::Empty),
+                DataType::DateTimeIso(_v) => {
+                    result_row.push(CellValue::DateTime(value.as_datetime().unwrap()))
+                }
+                DataType::DurationIso(_v) => {
+                    result_row.push(CellValue::DateTime(value.as_datetime().unwrap()))
```

Comment:
`fn as_datetime` always returns None for `DurationIso` (which represents time in ods).